### ISSUE TITLE
Can't find manifest

### ISSUE THREAD

**Issue by FireIceAce** [MicrosoftDocs/minecraft-creator#799]
getting the error can't find manifest, but it's right where is is instructed to be... need some help

![Screenshot 2024-02-26 212552](https://github.com/MicrosoftDocs/minecraft-creator/assets/161401964/59e3180e-5ac6-4ef5-b812-1ab55e472b66)
![Screenshot 2024-02-26 212708](https://github.com/MicrosoftDocs/minecraft-creator/assets/161401964/d4897891-9e37-49aa-aa98-02f887eef553)

![Screenshot 2024-02-26 212325](https://github.com/MicrosoftDocs/minecraft-creator/assets/161401964/9c4f0c29-cdbb-44ae-8ec2-f203737061f2)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 37ca30c2-bbe8-119e-3d74-05d344211a06
* Version Independent ID: 59430e36-a29d-ba99-38d3-ba78fc873d1f
* Content: [Create Custom Grass Blocks: An Introduction to Resource Packs](https://learn.microsoft.com/en-us/minecraft/creator/documents/resourcepack?view=minecraft-bedrock-stable)
* Content Source: [creator/Documents/ResourcePack.md](https://github.com/MicrosoftDocs/minecraft-creator/blob/main/creator/Documents/ResourcePack.md)
* Service: **minecraft-bedrock-edition**
* GitHub Login: @JimSeaman42
* Microsoft Alias: **mikeam**

**Comment by iconicNurdle** [comment]
@FireIceAce Would you please attach your manifest file here so I can take a look?

**Comment by FireIceAce** [comment]
Sure please see attached

**Comment by FireIceAce** [comment]
Hmm can't see if it got attached so let me try it from here
[manifest.json](https://github.com/MicrosoftDocs/minecraft-creator/files/14471843/manifest.json)

**Comment by iconicNurdle** [comment]
@FireIceAce When I download and open it, it's blank. Is the one on your side blank?

**Comment by FireIceAce** [comment]
No it's not so let me try again.
[manifest.json](https://github.com/MicrosoftDocs/minecraft-creator/files/14545159/manifest.json)

what I see when I open it.
![image](https://github.com/MicrosoftDocs/minecraft-creator/assets/161401964/3a2000df-61b5-4f6d-a935-07828dea41f8)

**Comment by iconicNurdle** [comment]
Can you get to this link: https://github.com/microsoft/minecraft-samples/tree/main/behavior_pack_sample

**Comment by iconicNurdle** [comment]
Hi. Do you still need help with this issue?

**Comment by iconicNurdle** [comment]
I am going to close this issue, but I'll be glad to help if I can get more information - specifically, are you able to get the sample behavior pack and/or resource pack to work? Let me know and we can go from there.
Thanks!
